sources/Interop/Windows/um/dxcapi/DXC_OUT_KIND.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/dxcapi.h in the Windows SDK for Windows 10.0.19041.0
```

Comment:
Shouldn't most of these headers have been updated? Either that or `other/dxc/dxcapi/generate.rsp` is missing some excludes